server/README.md

```diff
@@ -494,3 +511,20 @@ curl --location --request GET 'http://localhost:3333/api/v1/user'
     ]
 }
 ```
+
+## Get ws status (returns the status of ws connection mapped to the userID)
+
+### Example cURL Command:
+
+```
+curl --location --request GET 'http://localhost:3333/api/v1/status'
+```
+
+### 200 Response:
```

Comment:
Can we also give the 404 response if a user hasn't been found?

Reply:
Taken care